packages/lit-dev-content/site/tutorials/content/working-with-lists/04.md

```diff
@@ -0,0 +1,13 @@
+Congratulations on completing the **Working with Lists** tutorial!
```

Comment:
I think two more steps should be: events (talked on chat about that) and heterogenous lists. Since Lit just support iterables of any renderable value, the template results in an array don't have to be the same. So you can conditionally render different templates.